python/app/model_entity_listing.py

```diff
@@ -136,7 +141,7 @@ def load_data(
         # is the case when a script key is used for auth and we can't
         # determine a Shotgun human user by other means.
         try:
-            filters = self._get_filters()
+            filters = self._get_filters() + filters
```

Comment:
This will error if filters are not set and are `None`. I'd suggest ensuring filters are initialized to an empty list, e.g.:
```
filters = filters or []
```